Description: Upstream changes introduced in version 3.9.1.0-4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libzorpll (3.9.1.0-4) unstable; urgency=low
 .
   * Fixed a compilation error in some architectures.
 .
 The person named in the Author field signed this changelog entry.
Author: SZALAY Attila <sasa@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- libzorpll-3.9.1.0.orig/src/zorp/io.h
+++ libzorpll-3.9.1.0/src/zorp/io.h
@@ -34,9 +34,9 @@ void z_fd_set_our_tos(gint fd, guint8 to
 
 #else
 
-#define z_fd_get_peer_tos(fd, tos)
-#define z_fd_get_our_tos(fd, tos)
-#define z_fd_set_our_tos(fd, tos)
+#define z_fd_get_peer_tos(fd, tos) {}
+#define z_fd_get_our_tos(fd, tos) {}
+#define z_fd_set_our_tos(fd, tos) {}
 
 #endif
 
